warnings.rb

```diff
@@ -0,0 +1,10 @@
+
+a = { "hello" => "world" }
+
+2 + 5
```

Comment:
_[farcy v0.1b](https://github.com/appfolio/farcy)_

* Lint/Void: Operator `+` used in void context.